infrastructure/k8s/services/github.yaml

```diff
@@ -0,0 +1,39 @@
+project_name: "taskcluster-github"
+secrets:
+  azure_account: ".Values.tfAzureAccount"
+  force_ssl: "false"
+  github_app_id: ".Values.githubAppId" 
+  github_private_pem: ".Values.githubPrivatePem"
```

Comment:
Delete these from https://github.com/sciurus/taskcluster/blob/cloudops/infrastructure/terraform/to_review/more_variables.tf#L6-L19 since they aren't used in terraform anymore.